manifest.yml

```diff
@@ -46,3 +48,21 @@ applications:
         instances: ((worker_instances))
         command: yarn start:worker
         memory: ((worker_memory))
+  - name: tta-similarity-api-((env))
+    strategy: rolling
+    stack: cflinuxfs4
+    health-check-type: process
+    buildpacks:
+      - python_buildpack
+    path: similarity_api/src
+    services:
+      - ttahub-((env))
+    env:
+      NEW_RELIC_APP_NAME: tta-similarity-api-((env))
+      NEW_RELIC_LICENSE_KEY: ((NEW_RELIC_LICENSE_KEY))
```

Comment:
We still need to implement the `X-API-KEY` secret to be secret which should be unique per environment, including adding the vars to CircleCI.

Reply:
I will add these to CI and all ☁️.gov envs

Reply:
This is done